models/maker/ethereum/maker_ethereum_accounting.sql

```diff
@@ -146,6 +146,7 @@ WITH dao_wallet AS (
         ('RWA008-A',CAST(NULL as date),CAST(NULL as date),12310,31170,CAST(NULL AS NUMERIC(38))),
         ('RWA009-A',CAST(NULL as date),CAST(NULL as date),12310,31170,CAST(NULL AS NUMERIC(38))),
         ('RWA014-A',CAST(NULL as date),CAST(NULL as date),13411,31180,CAST(NULL AS NUMERIC(38))),
+        ('RWA015-A',CAST(NULL as date),CAST(NULL as date),12320,31172,CAST(NULL AS NUMERIC(38))),
```

Comment:
@web3lyt you will need to remove modifications to this file unless you migrate the entire spell to dunesql, otherwise CI will yell at us to do so and fail

Reply:
> @web3lyt you will need to remove modifications to this file unless you migrate the entire spell to dunesql, otherwise CI will yell at us to do so and fail

Gotcha, I figured this might be the case. I created a different PR only changing the legacy file at the following link and will close this PR: https://github.com/duneanalytics/spellbook/pull/4275